### DISCUSSION TITLE
Mobile webpage monitoring eip6963: announceProvider does not respond, I have connected to metamask

### DISCUSSION THREAD

**Discussion by hengshanMWC** [wevm/viem#2157] [Question]
It works on the computer website, but not on the mobile phone, causing an error.：undefined is not an object (evaluating 'provider.request')
<img width="624" alt="image" src="https://github.com/wevm/viem/assets/45714422/582a9701-cc60-4ee3-aed1-eb3138606944">

**Comment by tmm** [comment]
Unless you have a EIP-6963-enabled wallet hooked up to your mobile device, this won't work on mobile.
